src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -953,18 +953,51 @@ void MFEMSidreDataCollection::Load(const std::string& path,
   }
 }
 
-void MFEMSidreDataCollection::LoadExternalData(const std::string& path)
+void MFEMSidreDataCollection::LoadExternalData(const std::string& filename,
+                                               const std::string& group_name)
 {
+  // Use the user-provided group name or the DataCollection's base group
+  Group* grp = m_bp_grp->getDataStore()->getRoot();
+  if(!group_name.empty())
+  {
+  #if defined(AXOM_USE_MPI) && defined(MFEM_USE_MPI)
+    if(m_comm != MPI_COMM_NULL)
+    {
+      SLIC_ERROR(
+        "Loading external data with a group name is not supported in "
+        "parallel.");
```

Comment:
Will this be resolved by https://github.com/LLNL/axom/issues/1479 ?

Reply:
Yes, that is the intention.

Reply:
I put this in because I kept forgetting and getting no loading and feedback. So I put in an error to warn the user until @nselliott adds the new capability.